mypy/main.py

```diff
@@ -540,31 +537,52 @@ def get_init_file(dir: str) -> Optional[str]:
     'almost_silent': bool,
 }
 
+SHARED_CONFIG_FILES = ('setup.cfg', )
+
 
-def parse_config_file(options: Options, filename: str) -> None:
+def parse_config_file(options: Options, filename: Optional[str]) -> None:
     """Parse a config file into an Options object.
 
     Errors are written to stderr but are not fatal.
+
+    If filename isn't provided, fall back to default config file and then
+    to common shared files (currently setup.cfg).
```

Comment:
I would just say "to setup.cfg". If and when we change it we can update the comment.